cranelift/codegen/src/machinst/blockorder.rs

```diff
@@ -211,6 +215,11 @@ impl BlockLoweringOrder {
             }
         }
 
+        // Arbitrarily reorder the basic blocks, which shoud have no impact
```

Comment:
s/shoud/should/

Also let's reword this slightly to something like "If chaos-mode is enabled in the control plane, arbitrarily reorder ..." just to emphasize that `shuffle` is a no-op in the common (normal/release build) case.